src/install/java/frameworkInstallation/jboss.mdx

```diff
@@ -61,7 +61,7 @@ To pass the `-javaagent` argument on JBoss:
             <tbody>
                 <tr>
                 <td>
-                    Unix / macOS with 6.x EAP or 7.0.x AS and above
+                    Unix and macOS with 6.x EAP or 7.0.x AS and above
```

Comment:
@austin-schaefer I think these should be "or" because "and" implies you need both Unix and macOS

Reply:
Thanks, swapped these around!